include/groonga/groonga.h

```diff
@@ -560,21 +560,57 @@ typedef uint32_t grn_column_flags;
 
 /* flags for grn_obj_flags and grn_column_flags */
 
+/// Mask for column types.
+/// Used to isolate the column type bits from the flags.
 #define GRN_OBJ_COLUMN_TYPE_MASK (0x07)
-#define GRN_OBJ_COLUMN_SCALAR    (0x00)
-#define GRN_OBJ_COLUMN_VECTOR    (0x01)
-#define GRN_OBJ_COLUMN_INDEX     (0x02)
-
-#define GRN_OBJ_COMPRESS_MASK    (0x07 << 4)
-#define GRN_OBJ_COMPRESS_NONE    (0x00 << 4)
-#define GRN_OBJ_COMPRESS_ZLIB    (0x01 << 4)
-#define GRN_OBJ_COMPRESS_LZ4     (0x02 << 4)
+/// Scalar column type.
+/// Indicates that the column stores a scalar value.
+#define GRN_OBJ_COLUMN_SCALAR (0x00)
+/// Vector column type.
+/// Indicates that the column stores a vector of values.
+#define GRN_OBJ_COLUMN_VECTOR (0x01)
+/// Index column type.
+/// Indicates that the column is an inverted index.
+#define GRN_OBJ_COLUMN_INDEX  (0x02)
+
+#define GRN_OBJ_COMPRESS_MASK (0x07 << 4)
+#define GRN_OBJ_COMPRESS_NONE (0x00 << 4)
+/// zlib compression.
+/// Indicates that the objects are compressed using zlib.
+#define GRN_OBJ_COMPRESS_ZLIB (0x01 << 4)
+/// LZ4 compression.
+/// Indicates that the objects are compressed using LZ4.
+#define GRN_OBJ_COMPRESS_LZ4 (0x02 << 4)
 /* Just for backward compatibility. We'll remove it at 5.0.0. */
-#define GRN_OBJ_COMPRESS_LZO           GRN_OBJ_COMPRESS_LZ4
-#define GRN_OBJ_COMPRESS_ZSTD          (0x03 << 4)
+#define GRN_OBJ_COMPRESS_LZO GRN_OBJ_COMPRESS_LZ4
+/// Zstandard (zstd) compression.
+/// Indicates that the objects are compressed using Zstandard (zstd).
```

Comment:
1行目で`(zstd)`しているからもういいかな。

```suggestion
/// Indicates that the objects are compressed using Zstandard.
```

Reply:
fix: [60efac0](https://github.com/groonga/groonga/pull/2070/commits/60efac000583ce36ffc13bcec5bdd0083a9dc0b0)